packages/api/test/metrics.spec.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env mocha, browser */
+import assert from 'assert'
+import { endpoint } from './scripts/constants.js'
+
+describe('GET /metrics', () => {
+  it('retrieves metrics', async () => {
+    const res = await fetch(new URL('metrics/', endpoint).toString())
```

Comment:
```suggestion
    const res = await fetch(new URL('metrics/', endpoint))
```

> const fetchResponsePromise = fetch(resource [, init])
> Parameters
> `resource`
>     This defines the resource that you wish to fetch. This can either be: A string or any other object with a stringifier — including a URL object — that provides the URL of the resource you want to fetch.
– https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch

Reply:
👌🏼  There are other tests with the same pattern here, I will also update them